tests/projects/local-kit/kits/core-kit/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "hello-kit"
+name = "core-kit"
```

Comment:
Are we still planning on modifying the Cargo.toml file for Kits, My impression was that twoliter would have its own project toml that would also include kit config stuff.